src/containers/ImportResources/ImportResources.js

```diff
@@ -0,0 +1,176 @@
+    const payload = {
+      pipelinename,
+      serviceaccount,
+      gitresourcename,
+      gitcommit,
+      repourl,
+      applydirectory
+    };
+
+    if (repourl === '') {
+      this.setState({
+        invalidInput: true
+      });
+      return;
+    }
+
+    this.setState({
+      invalidInput: false
+    });
+
+    const promise = createPipelineRun(payload, namespace);
+    promise
+      .then(headers => {
+        const logsURL = headers.get('Content-Location');
+        const pipelineRunName = logsURL.substring(logsURL.lastIndexOf('/') + 1);
+        const finalURL = '#/pipelines/pipeline0/runs/'.concat(pipelineRunName);
+        this.setState({
+          logsURL: finalURL,
+          submitSuccess: true,
+          invalidInput: false
+        });
+      })
+      .catch(error => {
+        const statusCode = error.response.status;
+        switch (statusCode) {
+          case 500:
+            this.setState({
+              submitSuccess: false,
+              invalidInput: true
+            });
+            break;
+          default:
+        }
+      });
+    event.preventDefault();
+  };
+
+  render() {
+    return (
+      <div className="outer">
+        <h1 className="ImportHeader">
+          Import Tekton resources from repository
+        </h1>
+        <Form>
+          <TextInput
+            required
+            type="URL"
+            invalid={this.state.invalidInput}
+            invalidText="Please submit a valid URL"
+            helperText="The location of the YAML definitions to be applied (Git URL's supported)"
+            labelText="Repository URL"
+            placeholder="Enter repository URL"
+            data-testid="repository-url-field"
+            name="repositoryURL"
+            value={this.state.repositoryURL}
+            onChange={this.handleTextInput}
+          />
+          <TextInput
+            helperText="The path from which resources will be applied at the
+          specifed repository"
+            labelText="Repository directory (optional)"
+            placeholder="Enter repository directory"
+            data-testid="directory-field"
+            name="directory"
+            value={this.state.directory}
+            onChange={this.handleTextInput}
+          />
+          <ServiceAccountsDropdown
+            helperText="The SA that the PipelineRun applying resources will
+          run under"
+            titleText="Service Account (optional)"
+            className="saDropdown"
+            onChange={this.handleServiceAccount}
+          />
+          <Button kind="primary" onClick={this.handleSubmit}>
+            Import and Apply
+          </Button>
+          {this.state.submitSuccess && (
+            <ToastNotification
+              kind="success"
+              title="Triggered PipelineRun to apply Tekton resources"
+              subtitle=""
+              caption={
+                <a href={this.state.logsURL}> View status of this run </a>
```

Comment:
This still needs to be addressed https://github.com/tektoncd/dashboard/pull/182#discussion_r288507779